CANbus_Bootloader.c

```diff
@@ -0,0 +1,273 @@
+/** \file
```

Comment:
If you want to include this, could you put this in a subdirectory, with a comment up top on how to use it?